pkg/email/config.go

```diff
@@ -20,25 +20,32 @@ import (
 
 	"firebase.google.com/go/auth"
 	"github.com/google/exposure-notifications-server/pkg/secrets"
+	"github.com/google/exposure-notifications-verification-server/pkg/render"
 )
 
 // ProviderType represents a type of email provider.
 type ProviderType string
 
 const (
-	ProviderTypeNoop     ProviderType = "NOOP"
+	// ProviderTypeNoop is a no-op provider
+	ProviderTypeNoop ProviderType = "NOOP"
+
+	// ProviderTypeFirebase falls back to firebase's default email template.
+	// it uses password-reset rather than a true invitation.
 	ProviderTypeFirebase ProviderType = "FIREBASE"
-	ProviderTypeSMTP     ProviderType = "SIMPLE_SMTP"
+
+	// ProviderTypeSMTP composes emails and sends them via an external SMTP server.
+	ProviderTypeSMTP ProviderType = "SIMPLE_SMTP"
 )
 
 // Config represents the env var based configuration for email SMTP server connection.
 type Config struct {
 	ProviderType ProviderType
 
-	User     string `env:"EMAIL_USER" json:",omitempty"`
-	Password string `env:"EMAIL_PASSWORD" json:",omitempty"`
-	SMTPHost string `env:"EMAIL_SMTP_HOST" json:",omitempty"`
-	SMTPPort string `env:"EMAIL_SMTP_PORT" json:",omitempty"`
+	User     string `env:"EMAIL_USER"`
+	Password string `env:"EMAIL_PASSWORD"`
+	SMTPHost string `env:"EMAIL_SMTP_HOST"`
+	SMTPPort string `env:"EMAIL_SMTP_PORT, default=587"`
```

Comment:
It might be worth commenting (somewhere) that port 25 is blocked on GCP and most other places.